### DISCUSSION TITLE
Provided the required information, applied for several months or failed

### DISCUSSION THREAD

**Discussion by yg-yanggao** [community/community#58598] [GitHub Education]
### Hi everyone,

I bound my school's account, submitted the Xuexin network（Official documents of the Ministry of Education） or student ID card in the campus network environment, but the application has been unsuccessful, it has been several months, I am sad

**Comment by yg-yanggao** [comment]
i means edu-email

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
